pymc4/gp/gp.py

```diff
@@ -0,0 +1,213 @@
+"""
+Gaussian Process Models present in PyMC4's Gaussian Process module.
+
+"""
+
+
+import tensorflow as tf
+from .cov import ExpQuad
+from .mean import Zero
+from ..distributions import MvNormal, Normal
+from .util import stabilize
+
+
+__all__ = ["LatentGP"]
+
+
+class BaseGP:
+    def __init__(self, mean_fn=Zero(1), cov_fn=ExpQuad(1.0, 1.0, 1)):
```

Comment:
Not necessarily opposed to making a zero mean function the default, but there isnt much benefit to assume a default covariance function.